multigeiger/multigeiger.ino

```diff
@@ -183,11 +209,11 @@ volatile unsigned long isr_count_timestamp_2send= micros();
          bool          displayIsClear         = false;
          char          ssid[30];
          int           haveBME280             = 0;
-         float         t                      = 0.0;
-         float         h                      = 0.0;
-         float         p                      = 0.0;
+         float         bme_temperature        = 0.0;
+         float         bme_humidity           = 0.0;
+         float         bme_pressure           = 0.0;
```

Comment:
in case we would ever support different sensors for this, it would be better to name this just temperature, humidity, pressure.

Reply:
note that there is no need to have the sensor name in the variable name.
not for this temp. sensor and not for temp. sensors.

besides precision, the temperature measured should be always the same.
and if you send the measuref data, you need to read some variable there, that is easier if it is just 1 variable `temperature`.

Reply:
BTW, it's a totally different story for the geiger tube and CPM.

CPM makes only sense IF the tube is also known, because there are more sensitive tubes (higher CPM) and less sensitive.

But still we always call it radiation_cpm and just have the tube type elsewhere.